test/rulesets/LinearAlgebra/dense.jl

```diff
@@ -1,118 +1,115 @@
 @testset "linalg" begin
     @testset "dot" begin
         @testset "Vector" begin
-            rng, M = MersenneTwister(123456), 3
-            x, y = randn(rng, M), randn(rng, M)
-            ẋ, ẏ = randn(rng, M), randn(rng, M)
-            x̄, ȳ = randn(rng, M), randn(rng, M)
+            M = 3
+            x, y = randn(M), randn(M)
+            ẋ, ẏ = randn(M), randn(M)
+            x̄, ȳ = randn(M), randn(M)
             frule_test(dot, (x, ẋ), (y, ẏ))
-            rrule_test(dot, randn(rng), (x, x̄), (y, ȳ))
+            rrule_test(dot, randn(), (x, x̄), (y, ȳ))
         end
         @testset "Matrix" begin
-            rng, M, N = MersenneTwister(123456), 3, 4
-            x, y = randn(rng, M, N), randn(rng, M, N)
-            ẋ, ẏ = randn(rng, M, N), randn(rng, M, N)
-            x̄, ȳ = randn(rng, M, N), randn(rng, M, N)
+            M, N = 3, 4
+            x, y = randn(M, N), randn(M, N)
+            ẋ, ẏ = randn(M, N), randn(M, N)
+            x̄, ȳ = randn(M, N), randn(M, N)
             frule_test(dot, (x, ẋ), (y, ẏ))
-            rrule_test(dot, randn(rng), (x, x̄), (y, ȳ))
+            rrule_test(dot, randn(), (x, x̄), (y, ȳ))
         end
         @testset "Array{T, 3}" begin
-            rng, M, N, P = MersenneTwister(123456), 3, 4, 5
-            x, y = randn(rng, M, N, P), randn(rng, M, N, P)
-            ẋ, ẏ = randn(rng, M, N, P), randn(rng, M, N, P)
-            x̄, ȳ = randn(rng, M, N, P), randn(rng, M, N, P)
+            M, N, P = 3, 4, 5
+            x, y = randn(M, N, P), randn(M, N, P)
+            ẋ, ẏ = randn(M, N, P), randn(M, N, P)
+            x̄, ȳ = randn(M, N, P), randn(M, N, P)
             frule_test(dot, (x, ẋ), (y, ẏ))
-            rrule_test(dot, randn(rng), (x, x̄), (y, ȳ))
+            rrule_test(dot, randn(), (x, x̄), (y, ȳ))
         end
     end
     @testset "inv" begin
-        rng, N = MersenneTwister(123456), 3
-        B = generate_well_conditioned_matrix(rng, N)
-        frule_test(inv, (B, randn(rng, N, N)))
-        rrule_test(inv, randn(rng, N, N), (B, randn(rng, N, N)))
+        N = 3
+        B = generate_well_conditioned_matrix(Random.GLOBAL_RNG, N)
```

Comment:
we should fix this in ChainrulesTestUtils https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/27#issuecomment-621417499

Reply:
Yeah that makes more sense! I'll make the changes and re-open that CR and we can release that version and make the necessary changes in here.